tests/google/adwords.py

```diff
@@ -68,3 +68,67 @@ def test_adwords_data_upload_for_offline_conversion(self):
         print(uploaded, failed)
         self.assertEqual(len(uploaded), 2)
         self.assertEqual(len(failed), 1)
+
```

Comment:
I'd rather use "Should_ExpectedBehavior_When_StateUnderTest" naming convention for tests.